design-documents/graph-ql/coverage/catalog.graphqls

```diff
@@ -320,8 +320,12 @@ type Products @doc(description: "The Products object is the top-level object ret
     sort_fields: SortFields @doc(description: "An object that includes the default sort field and all available sort fields.") @resolver(class: "Magento\\CatalogGraphQl\\Model\\Resolver\\Category\\SortFields")
 }
 
-input AggregationsFilterInput {
-    includeSubcategoriesOnly: Boolean = false @doc(description: "Flag to include only subcategories of requested category.")
+input AggregationsFilterInput @doc(description: "Filter aggregations in layered navigation.") {
+    category: AggregationsCategoryFilterInput @doc(description: "Filter category aggregations in layered navigation.")
+}
+
+input AggregationsCategoryFilterInput @doc(description: "Filter category aggregations in layered navigation."){
+    includeDirectChildrenOnly: Boolean = false @doc(description: "Flag to include only direct subcategories of requested category.")
```

Comment:
Don't use the word flag. The AggregationsCategoryFilterInput represents an object of a parameter or an argument, and includeDirectChildrenOnly is a field in that argument object.

- Indicates whether to include only direct subcategories or all children categories at all levels.

look at this PR to gain more insight about wording for inputs:
https://github.com/magento-honey-badgers/magento2ce/pull/223/files

Reply:
Done